src/pudl/extract/eia860.py

```diff
@@ -106,10 +105,8 @@ def extract_eia860(context, raw_eia860__all_dfs):
     ds = context.resources.datastore
 
     if eia_settings.eia860.eia860m:
-        eia860m_data_source = DataSource.from_id("eia860m")
-        eia860m_date = eia860m_data_source.working_partitions["year_month"]
         eia860m_raw_dfs = pudl.extract.eia860m.Extractor(ds).extract(
-            year_month=eia860m_date
+            year_month=eia_settings.eia860.eia860m_date
         )
         raw_eia860__all_dfs = pudl.extract.eia860m.append_eia860m(
             eia860_raw_dfs=raw_eia860__all_dfs, eia860m_raw_dfs=eia860m_raw_dfs
```

Comment:
instead of re-extracting the one-month 860m for the normal 860 assets, we could now rely on the raw 860m assets. But rn they are not generated iteratively, but rather in one big clump which takes a while because there are a loooot of `year_month`s. So making the normal raw 860 assets depend on the new raw 860m assets would mean effectively delaying when we run 860. So i didn't do this. also it wouldn't save that much time bc one year of 860m is nbd. but idk @bendnorman

Reply:
That seems fine as long as we create an issue to track the extraction improvments.

Reply:
okay new issue created #3366